app/schema/api/current/models.rb

```diff
@@ -112,6 +113,7 @@ def blood_pressure
           deleted_at: { '$ref' => '#/definitions/nullable_timestamp' },
           created_at: { '$ref' => '#/definitions/timestamp' },
           updated_at: { '$ref' => '#/definitions/timestamp' },
+          recorded_at: { '$ref' => '#/definitions/timestamp' },
```

Comment:
Same here.

Reply:
can't, because existing v3 apps are not sending it.